src/index.html

```diff
@@ -8,34 +8,46 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <div class="card" data-qa="card">
+    <div class="wrapper">
+      <div class="wrapper__card card" data-qa="card">
 
-      <div class="card__image">
-        <img src="./images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
-      </div>
+        <div class="card__image card-top">
+          <img
+            src="./images/imac.jpeg"
+            class="img"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+        </div>
 
-      <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
-      <div class="card__code">Product code: 195434</div>
-      <div class="card__rate">
+        <div class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card__code">Product code: 195434</div>
+        <div class="card__rate">
```

Comment:
3. [CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements

also relevant for lines 38-41